app/app.go

```diff
@@ -422,7 +423,7 @@ func NewEvmos(
 		AddRoute(paramproposal.RouterKey, params.NewParamChangeProposalHandler(app.ParamsKeeper)).
 		AddRoute(distrtypes.RouterKey, distr.NewCommunityPoolSpendProposalHandler(app.DistrKeeper)).
 		AddRoute(upgradetypes.RouterKey, upgrade.NewSoftwareUpgradeProposalHandler(app.UpgradeKeeper)).
-		AddRoute(ibchost.RouterKey, ibcclient.NewClientProposalHandler(app.IBCKeeper.ClientKeeper)).
+		AddRoute(ibcclienttypes.RouterKey, ibcclient.NewClientProposalHandler(app.IBCKeeper.ClientKeeper)).
```

Comment:
should we keep host too? I'm not sure what that does, but maybe we keep it.

Reply:
idk, other chains don't seem to have it:

- https://github.com/cosmos/gaia/blob/main/app/app.go#L398
- https://github.com/osmosis-labs/osmosis/blob/main/app/keepers/keepers.go#L347